docs/reference/symbols.rst

```diff
@@ -10,58 +10,31 @@ and mathematical operations.
 See :ref:`Symbols <intro_optimization_symbols>` for an introduction to
 working with symbols.
 
-Base Classes
-============
-
 .. currentmodule:: dwave.optimization.model
 
 .. _symbols_base_symbols:
 
-.. autoclass:: Symbol
+Symbol
+======
 
-The following :class:`~dwave.optimization.model.Symbol` methods
-are inherited by the :class:`~dwave.optimization.model.ArraySymbol`
-class and :ref:`model symbols <symbols_model_symbols>`.
+All symbols inherit from the :class:`Symbol` class and therefore inherit its
+methods.
 
-.. autosummary::
-    :toctree: generated/
+.. autoclass:: Symbol
+    :members:
+    :inherited-members:
+    :member-order: bysource
 
-    ~Symbol.equals
-    ~Symbol.has_state
-    ~Symbol.id
-    ~Symbol.iter_predecessors
-    ~Symbol.iter_successors
-    ~Symbol.maybe_equals
-    ~Symbol.reset_state
-    ~Symbol.shares_memory
-    ~Symbol.state_size
-    ~Symbol.topological_index
+ArraySymbol
+===========
 
-.. autoclass:: ArraySymbol
+Most mathematical symbols inherit from the :class:`ArraySymbol` class and
+therefore inherit its methods.
 
-The following :class:`~dwave.optimization.model.ArraySymbol` methods
-are inherited by the :ref:`model symbols <symbols_model_symbols>`.
-
-.. autosummary::
-    :toctree: generated/
-
-    ~ArraySymbol.all
-    ~ArraySymbol.any
-    ~ArraySymbol.copy
-    ~ArraySymbol.flatten
-    ~ArraySymbol.has_state
-    ~ArraySymbol.max
-    ~ArraySymbol.maybe_equals
-    ~ArraySymbol.min
-    ~ArraySymbol.ndim
-    ~ArraySymbol.prod
-    ~ArraySymbol.reshape
-    ~ArraySymbol.sum
-    ~ArraySymbol.shape
-    ~ArraySymbol.size
-    ~ArraySymbol.state
-    ~ArraySymbol.state_size
```

Comment:
Unrelated to this PR (it was [this one](https://github.com/dwavesystems/dwave-optimization/pull/228)): missing colon

![image](https://github.com/user-attachments/assets/7935804d-2a5b-4df9-862a-a5beb3f5f646)

Feel free to leave this for a separate PR

Reply:
Yeah, let's fix that separately.

Reply:
https://github.com/dwavesystems/dwave-optimization/pull/231